apps/core-app/src/main/modules/system/active-app.ts

```
@@ -57,6 +57,25 @@ function parseInteger(value: unknown): number | null {
   return Number.isFinite(normalized) ? normalized : null
 }
 
+function getCommandErrorCode(error: unknown): string | null {
+  if (!error || typeof error !== 'object') return null
```

ESLint

[error] Expect newline after if

(antfu/if-newline)